src/nexpy/gui/plotview.py

```diff
@@ -179,7 +179,7 @@ def make_active(event):
                 self.ydata = event.ydata
             elif event.button == 3:
                 hasattr(self, 'otab')
-                self.otab.home()
+                self.otab.home(autoscale=False)
```

Comment:
That is weird. Fortunately, the if statement is almost certainly redundant, since otab is defined a few lines later, but I guess I should reinstate it just in case. Thanks for checking.